src/foam/u2/view/OverlayActionListView.js

```diff
@@ -231,9 +231,14 @@ foam.CLASS({
 
       this.onDetach(() => { this.overlay_ && this.overlay_.remove(); });
 
+      // sub to actions from view
       self.obj?.sub(function() {
         self.overlay_.close();
       });
+      // sub to actions from view data
+      self.obj?.data?.sub(function() {
```

Comment:
Shouldnt this sub to the slot?

Reply:
the slot sub doesn't work, what's the reason it should sub to the slot instead?

Reply:
The object doesnt change, the slot is what would change, what is this trying to fix?

Reply:
we don't need to worry about the obj changing, we just need to know when an action on the object has been called

Reply:
this is to fix actions from the obj on the overlay not triggering the overlay close, made one more update